samples/word/10-content-controls/insert-and-change-tracking-state.yaml

```diff
@@ -0,0 +1,124 @@
+order: 9
+id: word-content-controls-insert-and-change-tracking-state
+name: Get content controls by their change tracking state
+description: Inserts content controls then retrieves them by their change tracking state.
+author: xiruatms
+host: WORD
+api_set:
+    WordApi: '1.5'
+script:
+    content: |
+        $("#insert-controls").on("click", () => tryCatch(insertContentControls));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function insertContentControls() {
+          // Inserts content controls, changes them, then gets them by their change tracking state.
+          await Word.run(async (context) => {
+            let trackAddedArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.added];
+            let trackDeletedArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.deleted];
+            let trackNormalArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.normal];
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load();
+            await context.sync();
+            paragraphs.items[0].insertContentControl("PlainText");
+            paragraphs.items[1].insertContentControl("PlainText");
+            paragraphs.items[2].insertContentControl("RichText");
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.trackAll;
+            paragraphs.items[3].insertContentControl("RichText");
```

Comment:
Is there some significance to these controls being RichText?

Reply:
Not really. Just showing that they don't only apply to one type of content control.